geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/ImportDataFunction.java

```diff
@@ -40,9 +40,13 @@ public void execute(FunctionContext context) {
     final Object[] args = (Object[]) context.getArguments();
     final String regionName = (String) args[0];
     final String importFileName = (String) args[1];
-    boolean invokeCallbacks = false;
+    boolean parallel = false;
     if (args.length > 2) {
```

Comment:
is the args length check necessary? would caller present different args at different times?